lib/aws/ses/base.rb

```diff
@@ -42,7 +42,9 @@ module SES
     DEFAULT_HOST = 'email.us-east-1.amazonaws.com'
 
     DEFAULT_MESSAGE_ID_DOMAIN = 'email.amazonses.com'
-    
+
+    DEFAULT_SIGNATURE_VERSION = 4
```

Comment:
Dev from AWS SES here. Thanks for this initiative. Sigv4 is not just going to be the default, it's going to be the only signing method supported in the near future, so I don't think backward compatibility applies here, only Sigv4 can remain